File: c/solidRect.c
#include<stdio.h>
void Pattern(int iRow, int iCol)
{
   for(int i=0;i<=iRow;i++)
   {
       for(int j=0;j<=iCol;j++)
       {
           printf("* ");
       }
       printf("\n");

   }
        

}
int main()
{
int iValue1 = 0, iValue2 = 0;
printf("Enter number of rows and columns");
scanf("%d %d",&iValue1, &iValue2);
Pattern(iValue1, iValue2);
return 0;
}

File: solidRect.py
def solidRect(irow,icol):
  for ircnt in range(0,irow):
    for iccnt in range(0,icol):
      print("*",end="\t")
    print("\r")
def main():
  print("This is solid rectangle")
  irow=input("enter number of rows ")
  icol=input("enter number of columns")
  irow=int(irow)
  icol=int(icol)
  solidRect(irow,icol)
 
if __name__== "__main__":
  main()